### DISCUSSION TITLE
Support for z/OS Open Tools git port?

### DISCUSSION THREAD

**Discussion by lbdyck** [IBM/dbb#209] [Ideas]
Do y'all support the z/OS Open Tools `git` port?  That isn't indicated in the README.md.

**Comment by dennis-behm** [comment]
@lbdyck apologies for getting back to you late. 

I think we don't mention any particular port of `git` that is required/recommend/supported. 

I just ran the zAppBuild test framework pipeline that we maintain internally with the **git-2.43.0** port provided by the z/OS Open Tools which also clones/builds a couple of different application. It clones via http as well as ssh. 

All tests passed successfully.
